internal/handlers/pg/msg_find.go

```diff
@@ -154,3 +169,65 @@ func (h *Handler) MsgFind(ctx context.Context, msg *wire.OpMsg) (*wire.OpMsg, er
 
 	return &reply, nil
 }
+
+func getMaxTimeMSParameter(document *types.Document) (int32, error) {
+	var maxTimeMS int32
+	maxTimeMSParam, err := common.GetOptionalParam(document, "maxTimeMS", maxTimeMS)
+	if err != nil {
+		var commonErr *common.Error
+		if errors.As(err, &commonErr) {
+			return 0, getErrorForInvalidMaxTimeMS(document)
+		}
+
+		return 0, err
+	}
+
+	if maxTimeMSParam < 0 {
+		return 0, common.NewErrorMsg(
+			common.ErrBadValue,
+			fmt.Sprintf("%v value for maxTimeMS is out of range", maxTimeMSParam),
+		)
+	}
+
+	return maxTimeMSParam, nil
+}
+
+func getErrorForInvalidMaxTimeMS(document *types.Document) error {
+	v, err := document.Get("maxTimeMS")
+	if err != nil {
+		return nil
+	}
+
+	switch maxTimeMS := v.(type) {
```

Comment:
For all the error messages in this `switch`: is the difference between them so significant? Can't we just say something like `maxTimeMS must be a non-negative integer (but %v provided)` to cover all the cases in one? 

Also, the texts of those errors don't seem to be consistent. Some of them tell what maxTimeMS must be, for example, `maxTimeMS must be X`. But others are `%v value for maxTimeMS is out of range`, which doesn't give any exact details about the possible range.

To be honest, I like the  `maxTimeMS must be X` approach more because it gives the exact definition of what it should be, and the person who sees the error doesn't need to guess what's wrong with their parameter.

Reply:
hey there, 
@ribaraka @rumyantseva 
you can use `AliasFromType()`  func to make it easier.  

Here is what I did in my PR for `$set` in `internal/handlers/common/update.go` file https://github.com/FerretDB/FerretDB/pull/634/commits/e274db0512c67bdf6413e20c74243af4569e6dbc

Reply:
`AliasFromType` was added in order to avoid unnecessary marshaling of `type.Document` in error messages. 
Here there are no documents in error messages.
However, it will decrease the complexity while the error code is the same and an error message will be still similar to mongo. 
So my opinion is that makes sense to use it here.

Reply:
Wouldn't it be better to rename `AliasFromType` func to `GetAliasFromType`? WDYT everyone?

Reply:
Function names should start with a verb. `alias` is not a verb.

Reply:
Does this mean that you confirm my point of view?

Reply:
yep 
but I suggest listening what other people say)

Reply:
Okay, so let's discuss let's how similar our messages should be with the messages returned by mongodb. 
We can do this here or on Monday's Open Office House meeting. 
About `AliasFromType` func I prefer `GetAliasFromType` instead.